mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +186,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # create fake head points from sphere, and scale to similar numbers as hsp
+    hsp = _get_ico_surface(5)['rr']
+    hsp /= 10
+
+    # read in raw data using spherical hsp, and extract new hsp
+    raw = read_raw_kit(sqd_path, mrk_path, elp_txt_path, hsp)
+    hsp_dec = np.array([dig['r'] for dig in raw.info['dig']])
+
+    # with 10242 points and decimation at 5 mm, hsp_dec should be over 5000
```

Comment:
It just so happens that decimating with current parameters (5 mm boxes) of the sphere with 10242 points goes down to 5000+ points. I wanted to have a check that would have caught the error that's being fixed here. Agreed it's not the best choice (I.e. it could fail if at some point we decide to change the resolution parameter) but it was the best I could think of.